src/components/edit-profile/index.test.tsx

```diff
@@ -94,11 +94,30 @@ describe('EditProfile', () => {
     expect(saveButton(wrapper).prop('isDisabled')).toEqual(false);
   });
 
+  it('renders dropdown with ownedZIDs with the primaryZID as the first option', () => {
+    featureFlags.allowEditPrimaryZID = true;
+
+    const wrapper = subject({
+      currentPrimaryZID: '0://jane:smith',
+      ownedZIDs: ['0://john:doe', '0://jane:smith', '0://jijitsu:kaezen'],
+    });
+
+    const dropdown = wrapper.find('SelectInput[label=""]');
```

Comment:
Is there more than one SelectInput at a time on this screen? I.E., do we need the `label` filter?

Reply:
we don't here, but i had to add it on the component side (`label = ''`), as it is a required field in the zUI component